event-publisher/src/test/resources/uk/gov/ons/ctp/common/event/PackageFixture.CaseEvent.json

```diff
@@ -17,6 +17,7 @@
         "collectionExerciseId": "3883af91-0052-4497-9805-3238544fcf8a",
         "invalid": true,
         "refusalReceived": "HARD_REFUSAL",
+        "caseRef": "occaecat quis qui",
```

Comment:
Minor, but it would be easier to check the structure of this event if the fields were in the same order as the example in the ssdc-shared-events repo. ie. caseRef would be better if it appeared just after caseRef, as per the example.